all/remote/pom.xml

```diff
@@ -114,34 +174,69 @@
                               <exclude>META-INF/LICENSE.txt</exclude>
                               <exclude>META-INF/NOTICE</exclude>
                               <exclude>META-INF/NOTICE.txt</exclude>
+                              <exclude>jboss-as-checkstyle/checkstyle.xml</exclude>
+                              <exclude>schema/*.xsd</exclude>
+                              <exclude>__redirected/*</exclude>
                            </excludes>
                         </filter>
                      </filters>
                      <createSourcesJar>true</createSourcesJar>
                      <relocations>
                         <relocation>
-                           <pattern>org.apache</pattern>
-                           <shadedPattern>infinispan.org.apache</shadedPattern>
+                           <pattern>org.apache.commons</pattern>
+                           <shadedPattern>infinispan.org.apache.commons</shadedPattern>
+                        </relocation>
+                        <relocation>
+                           <pattern>org.slf4j</pattern>
+                           <shadedPattern>infinispan.org.slf4j</shadedPattern>
                         </relocation>
                         <relocation>
                            <pattern>com.google</pattern>
                            <shadedPattern>infinispan.com.google</shadedPattern>
                         </relocation>
-                        <!-- Unfortunatelly because of https://issues.jboss.org/browse/ISPN-5903
-                             and https://bugzilla.redhat.com/show_bug.cgi?id=1266831 we cannot relocate
-                             logging module, otherwise some integration modules might miss it
-                             when both Uber jar as well as ISPN commons are on classpath
-                          -->
-                        <!--<relocation>
+                        <relocation>
+                           <pattern>org.xnio</pattern>
+                           <shadedPattern>infinispan.org.xnio</shadedPattern>
+                        </relocation>
+                        <relocation>
+                           <pattern>org.jboss.threads</pattern>
+                           <shadedPattern>infinispan.org.jboss.threads</shadedPattern>
+                        </relocation>
+                        <relocation>
+                           <pattern>org.jboss.remoting3</pattern>
+                           <shadedPattern>infinispan.org.jboss.remoting3</shadedPattern>
+                        </relocation>
+                        <relocation>
                            <pattern>org.jboss.logging</pattern>
```

Comment:
In order to fix CDI classloading we need to relocate JBoss Logging. 

However if anyone wants to use Spring + Uber Jars - he will need to use this workaround: https://bugzilla.redhat.com/show_bug.cgi?id=1266831#c7